visualizations/visuals.py

```diff
@@ -0,0 +1,69 @@
+# DATA ETHICS -------------------------------------------------------------------------------------------------
+# For the most part court records are public information.
+# Try to keep more private information off the records.
+# Possibly keep first names off, if people are that interested let them look online.
+#
+# Things to know
+#   a) obtain informed consent from potential research participants
+#              In most legal cases, court records are public. But be safe and double check.
+#
+#   b) minimise the risk of harm to participants
+#              The Human Rights First Asylum is not trying to put people at risk or bring them harm.
+#
+#   c) protect their anonymity and confidentiality
+#              Again. HRFA isn't there to put people at risk or bring them harm. However, it is possible
+#              that by putting their information on site it may cause them complications. Don't use pictures,
+#              FIRST and LAST names in conjunction, DOB information, as this may lead to legal complications for HRFA.
+#
+#   d) avoid using deceptive practices
+#
+#   e) give participants the right to withdraw from your research
+#              Note: bring this up to Shareholders. It's possible to gain animosity by not allowing someone to withdraw
+#              the information you are using about them. 
+#

```

Comment:
These are all fantastic points that should be addressed with the stakeholder before live deployment.